libs/go/athenzutils/hostname_test.go

```diff
@@ -0,0 +1,36 @@
+// Copyright 2021 Verizon Media, Inc.
+// Licensed under the terms of the Apache version 2.0 license. See LICENSE file for terms.
+
+package athenzutils
+
+import "testing"
+
+func TestExtractHostname(test *testing.T) {
+
+	tests := []struct {
+		name     string
+		certFile string
+		hostname string
+	}{
+		{"nocn", "data/no_cn_x509.cert", ""},
+		{"emailnouri", "data/valid_email_x509.cert", ""},
+		{
+			"urihostname",
+			"../../../servers/zts/src/test/resources/athenz.examples.uri-hostname-only.pem",
```

Comment:
let's just copy this cert into data folder to make each module independent. we shouldn't have test cases in one module depend on resources from another module.

Reply:
done